src/web/index.html.ejs

```diff
@@ -3,17 +3,17 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width" />
-    <meta name="description" content="Theming demo for Firefox Quantum and beyond." />
+    <meta name="description" content="<%= htmlWebpackPlugin.options.description %>" />
     <!-- Open Graph meta tags-->
-    <meta property="og:title" content="Firefox Themer" />
-    <meta property="og:url" content="https://mozilla.github.io/Themer/" />
+    <meta property="og:title" content="<%= htmlWebpackPlugin.options.title %>" />
+    <meta property="og:url" content="<%= htmlWebpackPlugin.options.homepage %>" />
     <meta property="og:type" content="website" />
     <!-- Twitter card meta tags-->
-    <meta name="twitter:card" content="themer" />
+    <meta name="twitter:card" content="summary" />
```

Comment:
Not sure "themer" is a valid card type here. I stole "summary" from the TestPilot site's `<meta>` tags.